pkg/kubelet/rkt/rkt.go

```diff
@@ -935,6 +942,12 @@ func (r *Runtime) preparePod(pod *api.Pod, pullSecrets []api.Secret) (string, *k
 		newUnitOption("Service", "KillMode", "mixed"),
 	}
 
+	if pod.Spec.SecurityContext != nil && pod.Spec.SecurityContext.SELinuxOptions != nil {
+		opt := pod.Spec.SecurityContext.SELinuxOptions
+		selinuxContext := fmt.Sprintf("%s:%s:%s:%s", opt.User, opt.Role, opt.Type, opt.Level)
```

Comment:
@yifan-gu you might want to project the specified fields onto the root directory SELinux context.